wiki/Forums/id.md

```diff
@@ -0,0 +1,76 @@
+# Forum
+
+![Forum osu!](main.jpg "Gambar dari forum osu!")
+
+Forum osu! merupakan salah satu fitur utama dari situs web osu!.
+Di sini, Anda dapat membuat topik atau kiriman baru dan berdiskusi bersama dalam komunitas osu!.
+
+## Subforum
+
+- [osu!](https://osu.ppy.sh/community/forums/1): _Bahasan seputar osu!_
+  - [Development](https://osu.ppy.sh/community/forums/2): _Ayo bawa osu! selangkah lebih maju. Diskusikan proyek, ide-ide baru, periklanan dan berita, rapat dan sebagainya._
+    - [Completed Projects](https://osu.ppy.sh/community/forums/116): _Proyek yang telah selesai dapat diarsipkan dan didiskusikan di sini._
+      - [osu! iPhone](https://osu.ppy.sh/community/forums/47)
+        - [Bug Reports](https://osu.ppy.sh/community/forums/50)
+    - [Community Voice](https://osu.ppy.sh/community/forums/65): _Jajak pendapat mingguan untuk membantu kami memahami komunitas kami sedikit lebih baik._
+    - [Ranking Criteria](https://osu.ppy.sh/community/forums/87)
+      - [Finalized/Denied Amendments](https://osu.ppy.sh/community/forums/92)
+    - [osu!droid](https://osu.ppy.sh/community/forums/85)
+      - [Bug Reports](https://osu.ppy.sh/community/forums/86)
+  - [Gameplay & Rankings](https://osu.ppy.sh/community/forums/13): _Diskusikan skor Anda, strategi bermain, dan lain-lain. Tantang pemain lain. Apa pun yang berhubungan dengan osu!_
+    - [Tournaments](https://osu.ppy.sh/community/forums/55)
+    - [Mapping Techniques](https://osu.ppy.sh/community/forums/61)
+    - [Taiko](https://osu.ppy.sh/community/forums/105)
+    - [Catch the Beat](https://osu.ppy.sh/community/forums/106)
+    - [osu!mania](https://osu.ppy.sh/community/forums/107)
+  - [Skinning](https://osu.ppy.sh/community/forums/15): _Diskusikan skinning dan kustomisasi!_
```

Comment:
add `tentang` between `diskusikan ... skinning`

will look like this

> _Diskusikan tentang skinning dan kustomisasi!_

Use `tentang` when the object is ver. When the object use nomina, no need to use `tentang` like line 20